common/changes/ianc-removing-deprectated-tslint-rules_2016-12-22-19-50.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/gulp-core-build-typescript",
+      "comment": "Fixing TSLint task by removing some deprecated rules (\"label-undefined”, \"no-duplicate-key\", and \"no-unreachable\") and setting the \"noUnusedParameters\" and \"noUnusedLocals\" TS compiler options to cover the deprecated \"no-unused-variable\".",
+      "type": "patch"
```

Comment:
We already did the version bump of TSLint, so this is currently broken. It should be a patch on version 2.0.0, not 2.1.0 or 3.0.0.